lab_3_ann_retriever/main.py

```diff
@@ -75,6 +99,13 @@ def load_vector(state: dict) -> Vector | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not (isinstance(state, dict) and "len" in state and "elements" in state and isinstance(
+            state["len"], int) and isinstance(state["elements"], dict)):
+        return None
+    vector = list(0.0 for _ in range(state["len"]))
+    for idx, value in state["elements"].items():
```

Comment:
Let's try to optimize it and use only one for to fill the vector

Reply:
done